app/main/views/users.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from flask import jsonify, request, abort
+from app.main import main
+from app.models import User
+from app import db, encryption
+from sqlalchemy.exc import IntegrityError
+from app.main import helpers
+from app.validation import validate_user_json_or_400, \
+    validate_user_auth_json_or_400
+
+
+@main.route('/users/auth', methods=['POST'])
+def auth_user():
+    json_payload = get_json_from_request('auth_users')
+
+    validate_user_auth_json_or_400(json_payload)
+
+    user = User.query.filter(
+        User.email_address == json_payload['email_address'].lower()).first()
+
+    if user is None:
+        return jsonify(authorization=False), 404
+    elif encryption.checkpw(json_payload['password'], user.password):
+        return jsonify(users=user.serialize()), 200
+    else:
+        return jsonify(authorization=False), 403
+
+
+@main.route('/users/<int:user_id>', methods=['GET'])
+def get_user_by_email(user_id):
+    user = User.query.filter(
+        User.id == user_id
+    ).first_or_404()
+    return jsonify(users=user.serialize())
+
+
+@main.route('/users', methods=['POST'])
+def create_user():
+    json_payload = get_json_from_request('users')
+    validate_user_json_or_400(json_payload)
+
+    user = User.query.filter(
+        User.email_address == 'joeblogs@email.com').first()
+
+    if user:
+        abort(409, "User already exists")
+
+    if 'hashpw' in json_payload and not json_payload['hashpw']:
+        password = json_payload['password']
```

Comment:
This hashpw field exists to get us over the hump of the initial import - the export from the exitsing DM is pre-hashed, once the import is complete this notion of "don't hash this password" will need to be removed. Default is hash, and the field is used only by the import script.